plugins/modules/win_gpo_force.ps1

```diff
@@ -0,0 +1,75 @@
+#!powershell
+# Copyright: (c) 2021 Sebastian Gruber (@sgruber94) ,dacoso GmbH All Rights Reserved.
+# SPDX-License-Identifier: GPL-3.0-only
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#AnsibleRequires -CSharpUtil Ansible.Basic
+
+$spec = @{
+    options             = @{
+        mode = @{ type = "str"; choices = "sysvolonly", "forceupdate"; required =$true}
+        ou   = @{ type = "str"; aliases = @("organizational_unit") }
+    }
+    supports_check_mode = $false
+}
+$module = [Ansible.Basic.AnsibleModule]::Create($args, $spec)
+$mode = $module.Params.mode
+$ou = $module.Params.ou
+try {
+    Import-Module GroupPolicy
+} catch {
+    $module.FailJson("win_gpo_force requires the GroupPolicy PS module to be installed")
+}
+function Invoke-forcedgpo {
+    param(
+        [string]$oucomputer	# Push gpupdate after AD & SYSVOL Replication
+    )
+    # Check for DFSR cmd tools
+    if ((Get-WindowsFeature -Name RSAT-DFS-Mgmt-Con).InstallState -ne "Installed") {
+        Install-WindowsFeature -Name RSAT-DFS-Mgmt-Con
+    }
+    # Check powershell console for UAC
+    #requires -RunAsAdministrator
+    # Force Replication of all Partitions between Domain Controllers
+    (Get-ADDomainController -filter *).Name | ForEach-Object { repadmin /syncall $_ (Get-ADDomain).DistinguishedName /e /A | Out-Null }
+    Start-Sleep 5
```

Comment:
Is there anyway to avoid this sleep. Sleeping is never a good idea as

1. It's never guaranteed to wait long enough, what if the host is slower than normal
2. It wastes time if what you are waiting for is already ready, i.e. it's ready in 1 second but the script still waits the additional 4.